Config/iOS.xcconfig

```diff
@@ -1,8 +1,17 @@
 SDK_NAME = appcenter.ios
 SDKROOT = iphoneos
-ARCHS = armv7 armv7s arm64
+PRE_XCODE_10_1_ARCHS = armv7 armv7s arm64
+ARCHS[sdk=iphoneos*] = $(PRE_XCODE_10_1_ARCHS) arm64e
+ARCHS[sdk=iphoneos9.*] = $(PRE_XCODE_10_1_ARCHS)
+ARCHS[sdk=iphoneos10.*] = $(PRE_XCODE_10_1_ARCHS)
+ARCHS[sdk=iphoneos11.*] = $(PRE_XCODE_10_1_ARCHS)
+ARCHS[sdk=iphoneos12.0] = $(PRE_XCODE_10_1_ARCHS)
 ARCHS[sdk=iphonesimulator*] = $(MS_SIM_ARCHS)
-VALID_ARCHS = $(ARCHS)
+VALID_ARCHS[sdk=iphoneos*] = $(ARCHS)
+VALID_ARCHS[sdk=iphoneos9.*] = $(ARCHS)
```

Comment:
Do you need line 11-14?

Reply:
yes, they are absolutely required.
Without specifying the versions individually, it will not work.